autopush/settings.py

```diff
@@ -204,18 +206,35 @@ def message(self, value):
         """Setter to set the current message table"""
         self.message_tables[self.current_msg_month] = value
 
+    def add_tomorrow(self, today, this_month_table):
```

Comment:
nit: so far we have last_month and this_month, maybe next_month and calling this add_next_month?